scripts/ci-cluster-provision.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
When does this file get called? Is this called from the cross-cloud-ci scripts itself somehow?